### PR TITLE
add VegasDreamworks.yml

### PR DESCRIPTION
**Pull request by JaseNZC**

### CODE REVIEW

File: scrapers/VegasDreamworks.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: VegasDreamworks
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - toticos.com
+      - paradisegfs.com
```
Comment (bnkai) [inline scrapers/VegasDreamworks.yml:6]: can you fix the order?

File: scrapers/VegasDreamworks.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@class="post-title"]
+      Performers:
+        Name: //div[@class="single-model-head"]/a | //div[@class="update_bb_info"]/a
```
Comment (bnkai) [inline scrapers/VegasDreamworks.yml:14]: the parent is the same in both sites a simple
```suggestion
        Name: //div[@class="update_bb_info"]//a
```
should be ok

**Comment by bnkai** [comment]
The details when the scenes have a lot of paragraphs(Read More...) seem a little bit off (only the appearance) but its not worth the effort to do something more imo.

**Comment by JaseNZC** [comment]
Yeah the details arn't the best but as you said not really worth doing any more to it.

**Pull request merged**